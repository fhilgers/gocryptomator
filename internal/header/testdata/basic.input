{
    "example": {
        "Header": "mDCOS8MT0etiyDKaXv8X0cD1ayTewk/Uir95AZO5t4bwy8sQm5yl23Y0XxyQtg43/IQJLyHLJSuVfKLEQuPmncqbbndSiMzU2iS00Mo9lM3/1XCmLJWmig==",
        "EncKey": "ABygFz/iZU0HXYlE5NtYFPgEmKqyWEh4wGjHo0fJgNY=",
        "MacKey": "lwdzbaZqIfibYj8LSjcUAqpOJfdku/qN13f2OZWpxDk="
    }
}
