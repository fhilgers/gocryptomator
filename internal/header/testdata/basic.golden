{
    "example": {
        "Nonce": "mDCOS8MT0etiyDKaXv8X0Q==",
        "Reserved": "//////////8=",
        "ContentKey": "RgbuFMgbZP051gzgXp8WbQNJ5y50eNyyBL/6qdQNn6c="
    }
}
